crates/ruff/src/rules/flake8_executable/snapshots/ruff__rules__flake8_executable__tests__EXE005_2.py.snap

```diff
@@ -1,11 +1,11 @@
 ---
 source: crates/ruff/src/rules/flake8_executable/mod.rs
 ---
-EXE005_2.py:4:3: EXE005 Shebang should be at the beginning of the file
+EXE005_2.py:4:1: EXE005 Shebang should be at the beginning of the file
   |
 3 | # A python comment
 4 | #!/usr/bin/python
-  |   ^^^^^^^^^^^^^^^ EXE005
+  | ^^^^^^^^^^^^^^^^^ EXE005
```

Comment:
This is intentional, I think we should include this in the diagnostic range.

Reply:
Agreed.